### PR TITLE
Component

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Add components from my Ecomm app to react
Components added:
1. Vertical Card
2. Product Filter
3. Cart Card
4. Footer
5. Header
6. Price Card

### CODE REVIEW

**Review by anandsr21harshit — APPROVED** [review]
Good folder structure and neat code

**Review by lomesshh — APPROVED** [review]
code looks modular
nice folder structure
looks good to merge

**Review by MurtazaBagwala07 — APPROVED** [review]
code is easy to understand
looks good to merge

File: src/components/ProductFilter/ProductFilter.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  padding: 0;
+  margin: 0;
+  text-decoration: underline;
+}
+
+.checkbox-container {
+  padding: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: normal;
```
Comment (Logan1x) [inline src/components/ProductFilter/ProductFilter.css:48]: How does this behave?

**Review by Logan1x — APPROVED** [review]

**Pull request merged**